### PR TITLE
WebAPI: fixed constructor section code

### PR DESCRIPTION
**Pull request by OnkarRuikar**
#### Summary
Related to #14114

Constructor syntax section doesn't need variable names.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/compositionevent/compositionevent/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ constructor creates a new {{domxref("CompositionEvent")}} object instance.
 ## Syntax
 
 ```js
-const myEvent = new CompositionEvent(typeArg [, CompositionEventInit])
+new CompositionEvent(typeArg [, CompositionEventInit])
```
Comment (sideshowbarker) [inline files/en-us/web/api/compositionevent/compositionevent/index.md:19]:
```suggestion
new CompositionEvent(typeArg)
new CompositionEvent(typeArg, CompositionEventInit)
```

File: files/en-us/web/api/devicemotionevent/devicemotionevent/index.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ The **`DeviceMotionEvent()`** constructor creates a new
 ## Syntax
 
 ```js
-var deviceMotionEvent = new DeviceMotionEvent(type[, options])
+new DeviceMotionEvent(type[, options])
```
Comment (sideshowbarker) [inline files/en-us/web/api/devicemotionevent/devicemotionevent/index.md:25]:
```suggestion
new DeviceMotionEvent(type)
new DeviceMotionEvent(type, options)
```

File: files/en-us/web/api/deviceorientationevent/deviceorientationevent/index.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ The **`DeviceOrientationEvent()`** constructor creates a new
 ## Syntax
 
 ```js
-var deviceOrientationEvent = new DeviceOrientationEvent(type[, options])
+new DeviceOrientationEvent(type[, options])
```
Comment (sideshowbarker) [inline files/en-us/web/api/deviceorientationevent/deviceorientationevent/index.md:21]:
```suggestion
new DeviceOrientationEvent(type)
new DeviceOrientationEvent(type, options)
```

File: files/en-us/web/api/dommatrix/dommatrix/index.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ operations.
 ## Syntax
 
 ```js
-var domMatrix = new DOMMatrix([init])
+new DOMMatrix([init])
```
Comment (sideshowbarker) [inline files/en-us/web/api/dommatrix/dommatrix/index.md:23]:
```suggestion
new DOMMatrix()
new DOMMatrix(init)
```

**Comment by OnkarRuikar** [comment]
@sideshowbarker thanks for the corrections!

File: files/en-us/web/api/extendablecookiechangeevent/extendablecookiechangeevent/index.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ The **`ExtendableCookieChangeEvent()`** constructor creates a new {{domxref("Ext
 ## Syntax
 
 ```js
-var extendableCookieChangeEvent = new ExtendableCookieChangeEvent(type, [eventInitDict]);
+new ExtendableCookieChangeEvent(type, [eventInitDict]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/extendablecookiechangeevent/extendablecookiechangeevent/index.md:20]:
```suggestion
new ExtendableCookieChangeEvent(type);
new ExtendableCookieChangeEvent(type, eventInitDict);
```

File: files/en-us/web/api/focusevent/focusevent/index.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ set to the other target.
 ## Syntax
 
 ```js
-var focusEvent = new FocusEvent(typeArg[, focusEventInit]);
+new FocusEvent(typeArg[, focusEventInit]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/focusevent/focusevent/index.md:22]:
```suggestion
new FocusEvent(typeArg);
new FocusEvent(typeArg, focusEventInit);
```

File: files/en-us/web/api/fontfacesetloadevent/fontfacesetloadevent/index.md [hunk 7]
```diff
@@ -21,7 +21,7 @@ The **`FontFaceSetLoadEvent()`** constructor creates a new
 ## Syntax
 
 ```js
-var fontFaceSetLoadEvent = new FontFaceSetLoadEvent(type[, options])
+new FontFaceSetLoadEvent(type[, options])
```
Comment (sideshowbarker) [inline files/en-us/web/api/fontfacesetloadevent/fontfacesetloadevent/index.md:24]:
```suggestion
new FontFaceSetLoadEvent(type)
new FontFaceSetLoadEvent(type, options)
```

File: files/en-us/web/api/inputdevicecapabilities/inputdevicecapabilities/index.md [hunk 8]
```diff
@@ -10,7 +10,7 @@ device responsible for generating a touch event.
 ## Syntax
 
 ```js
-var inputDeviceCapabilities = new InputDeviceCapabilities([InputDeviceCapabilitiesInit])
+new InputDeviceCapabilities([InputDeviceCapabilitiesInit])
```
Comment (sideshowbarker) [inline files/en-us/web/api/inputdevicecapabilities/inputdevicecapabilities/index.md:13]:
```suggestion
new InputDeviceCapabilities()
new InputDeviceCapabilities(InputDeviceCapabilitiesInit)
```

File: files/en-us/web/api/mediarecorder/mediarecorder/index.md [hunk 9]
```diff
@@ -26,7 +26,7 @@ and codec configuration(s) to use by specifying [the `codecs` parameter](/en-US/
 ## Syntax
 
 ```js
-var mediaRecorder = new MediaRecorder(stream[, options]);
+new MediaRecorder(stream[, options]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/mediarecorder/mediarecorder/index.md:29]:
```suggestion
new MediaRecorder(stream)
new MediaRecorder(stream, options)
```

File: files/en-us/web/api/pictureinpictureevent/pictureinpictureevent/index.md [hunk 10]
```diff
@@ -13,7 +13,7 @@ created {{domxref("PictureInPictureEvent")}} object with an optional
 ## Syntax
 
 ```js
-var pictureInPictureEvent = new PictureInPictureEvent(typeArg[, pictureInPictureInit]);
+new PictureInPictureEvent(typeArg[, pictureInPictureInit]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/pictureinpictureevent/pictureinpictureevent/index.md:16]:
```suggestion
new PictureInPictureEvent(typeArg)
new PictureInPictureEvent(typeArg, pictureInPictureInit);
```

File: files/en-us/web/api/readablestream/readablestream/index.md [hunk 11]
```diff
@@ -16,7 +16,7 @@ and returns a readable stream object from the given handlers.
 ## Syntax
 
 ```js
-var readableStream = new ReadableStream(underlyingSource[, queuingStrategy]);
+new ReadableStream(underlyingSource[, queuingStrategy]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/readablestream/readablestream/index.md:19]:
```suggestion
new ReadableStream(underlyingSource)
new ReadableStream(underlyingSource, queuingStrategy)
```

File: files/en-us/web/api/request/request/index.md [hunk 12]
```diff
@@ -17,7 +17,7 @@ The **`Request()`** constructor creates a new
 ## Syntax
 
 ```js
-var myRequest = new Request(input[, init]);
+new Request(input[, init]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/request/request/index.md:20]:
```suggestion
new Request(input)
new Request(input, init)
```

File: files/en-us/web/api/submitevent/submitevent/index.md [hunk 13]
```diff
@@ -24,7 +24,7 @@ represent a {{domxref("HTMLFormElement.submit_event", "submit")}} event fired at
 ## Syntax
 
 ```js
-let submitEvent = new SubmitEvent(type,eventInitDict);
+new SubmitEvent(type,eventInitDict);
```
Comment (sideshowbarker) [inline files/en-us/web/api/submitevent/submitevent/index.md:27]:
```suggestion
new SubmitEvent(type, eventInitDict);
```

File: files/en-us/web/api/writablestream/writablestream/index.md [hunk 14]
```diff
@@ -18,7 +18,7 @@ a new {{domxref("WritableStream")}} object instance.
 ## Syntax
 
 ```js
-var writableStream = new WritableStream(underlyingSink[, queuingStrategy]);
+new WritableStream(underlyingSink[, queuingStrategy]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/writablestream/writablestream/index.md:21]:
```suggestion
new WritableStream(underlyingSink)
new WritableStream(underlyingSink, queuingStrategy)
```

File: files/en-us/web/api/animation/animation/index.md [hunk 15]
```diff
@@ -17,7 +17,7 @@ The **`Animation()`** constructor of the [Web Animations API](/en-US/docs/Web/AP
 ## Syntax
 
 ```js
-var animation = new Animation([effect][, timeline]);
+new Animation([effect][, timeline]);
```
Comment (sideshowbarker) [inline files/en-us/web/api/animation/animation/index.md:20]:
```suggestion
new Animation()
new Animation(effect)
new Animation(effect, timeline)
```

File: files/en-us/web/api/mediametadata/mediametadata/index.md [hunk 16]
```diff
@@ -21,7 +21,7 @@ The **`MediaMetadata()`** constructor creates a new
 ## Syntax
 
 ```js
-var mediaMetadata = new MediaMetadata([metadata])
+new MediaMetadata([metadata])
```
Comment (OnkarRuikar) [inline files/en-us/web/api/mediametadata/mediametadata/index.md:24]:
```suggestion
new MediaMetadata()
new MediaMetadata(metadata)
```

File: files/en-us/web/api/url/url/index.md [hunk 17]
```diff
@@ -23,7 +23,7 @@ If the given base URL or the resulting URL are not valid URLs, the JavaScript
 ## Syntax
 
 ```js
-const url = new URL(url [, base])
+new URL(url [, base])
```
Comment (OnkarRuikar) [inline files/en-us/web/api/url/url/index.md:26]:
```suggestion
new URL(url)
new URL(url, base)
```

**Review by sideshowbarker — APPROVED** [review]
Thanks much for this — nice to be getting rid of the cruft

**Pull request merged**